src/main/java/aquality/selenium/locators/RelativeBy.java

```diff
@@ -0,0 +1,111 @@
+package aquality.selenium.locators;
+
+import aquality.selenium.elements.interfaces.IElement;
+import org.openqa.selenium.By;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.locators.RelativeLocator;
+import java.util.List;
+
+public class RelativeBy extends By implements IRelativeByAqualityElement, IRelativeByWebElement, IRelativeBy {
+    private By by;
+
+    public RelativeBy(By by) {
+        this.by = by;
+    }
+
+    public RelativeBy above(IElement element) {
+        by = RelativeLocator.with(by).above(element.getElement());
+        return new RelativeBy(by);
+    }
+
+    public RelativeBy above(WebElement element) {
+        by = RelativeLocator.with(by).above(element);
+        return new RelativeBy(by);
+    }
+
+    public RelativeBy above(By by) {
+        this.by = RelativeLocator.with(this.by).above(by);
+        return new RelativeBy(this.by);
+    }
+
+    public RelativeBy below(IElement element) {
+        by = RelativeLocator.with(by).below(element.getElement());
```

Comment:
I'm just wondering, could we make it kind of lazy initialization? E.g. If I will initialize a field, don't want to get related element at that moment

Reply:
Yes, If I understand right. We can write as such  by = RelativeLocator.with(by).below(element.getLocator()), using realization below(By by)
In this case we can avoid working with elementFinder using By reference object that is simple stored in IElement realization class.